### DISCUSSION TITLE
Nesting a Text/Number Input inside of Table Cell

### DISCUSSION THREAD

**Discussion by deltasierra96** [adobe/react-spectrum#6073] [Q&A] [answered]
As the title suggests, is it possible to nest text/number inputs in a table cell? I've tried and unfortunately when trying to focus on the element, it fails and selects the entire row. 

I can't seem to see anything regarding slots for these elements and I'm at a loss.

I'm trying to achieve something similar to [this](https://prnt.sc/p2xvudiWeaFq).

**Comment by reidbarber** [comment, marked as answer]
Not at the moment unfortunately, but it is on our roadmap. There is an issue for this here: https://github.com/adobe/react-spectrum/issues/2328

**Reply by deltasierra96** [reply]
I seen that but it was from 2022 and thought I was overlooking some new functionality. I can appreciate the complexity with these components so I'll check back another time.

Thanks, @reidbarber

**Comment by brilliant-ember** [comment]
yeah me too! I can't get my text box to focus/work inside the rows of a selectable-table. It's actually so not easy, I tried overriding the onClick props to noavail. 
For the time being, I disabled the table selection option and I am manually adding a checkbox column that only selects if the checkbox is clicked, and I keep track of the selection state with my own selection context. It's working but would be rad if we could get some direction on how to do it using React-Aria.
